### PR TITLE
[Hello] News infinite scroll

### PR DESCRIPTION
**Pull request by MysticFragilist**
### ⁉️ Related Issue

no issues related (Hello special project)

## 📖 Description

Added an infinite scroll in the news tab to handle the paginated endpoint.

### 🧪 How Has This Been Tested?

Added test and widget test. Simply navigate to news view and using an hello URL defined in ETS-API-Clients.

### ☑️ Checklist before requesting a review
- [x] I have performed a self-review of my code.
- [x] If it is a core feature, I have added thorough tests.
- [x] If needed, I added analytics.
- [x] Make sure to add either one of the following labels: `version: Major`,`version: Minor` or `version: Patch`. 
- [x] Make sure golden files changes were reviewed and approved.

### 🖼️ Screenshots (if useful):

image
image

### CODE REVIEW

File: l10n/intl_fr.arb [hunk 1]
```diff
@@ -171,6 +171,10 @@
   "ets_papercut_title":"PaperCut",
 
   "news_title" : "Annonces",
+  "news_error_not_found_title" : "Oh oh!",
+  "news_error_not_found" : "Une erreur est survenue lors de la récupération des annonces. Veuillez réessayer plus tard.",
+  "news_no_more_card_title": "Tu es à jours!",
```
Comment (Udito3) [inline l10n/intl_fr.arb:176]: Un petit "s" de trop à jour

**Review by Udito3 — APPROVED** [review]
Sauf la trad, LGTM!

**Comment by Udito3** [comment]
Ah ouais mais c'est pas dans la bonne branche que tu merge!

**Review by camillebrulotte — APPROVED** [review]
LGTM !

**Pull request merged**